src/src/App.tsx

```diff
@@ -0,0 +1,8 @@
+import { TodosProvider } from './services/Store';
+import { TodoApp } from './components/todoApp';
+
+export const App: React.FC = () => (
+  <TodosProvider>
+    <TodoApp />
+  </TodosProvider>
+);
```

Comment:
Duplicates, so need to fix it 
<img width="722" alt="image" src="https://github.com/mate-academy/react_todo-app-add-and-delete/assets/64750958/bed6f9f5-3dd2-478b-8230-211b63413d13">
<img width="354" alt="image" src="https://github.com/mate-academy/react_todo-app-add-and-delete/assets/64750958/dbb300f6-bedf-4133-be80-4200909d9bf1">